prisma/schema.prisma

```diff
@@ -132,8 +132,8 @@ model TeacherReg {
 
 model User {
   id            Int           @id @default(autoincrement())
-  firstName     String?       @map("first_name")
```

Comment:
Why did you change this?

Reply:
I'm not sure, when I ran commands I think it auto updated

Reply:
It should be `firstName` with the mapping. Is your schema.sql file up to date?